### DISCUSSION TITLE
Example Kubernetes deployments?

### DISCUSSION THREAD

**Discussion by SimplicityGuy** [qdm12/gluetun#618] [Q&A]
Looking through the Wiki, I did not see any example Kubernetes deploy specs. If anyone is using `gluetun` with Kubernetes, can you post your config here please?

**Comment by qdm12** [comment]
Hey there! Sorry I missed your discussion down my email inbox!

I don't personally use gluetun with Kubernetes, but it shouldn't be too hard to convert the docker-compose.yml of the readme into a kubernetes spec. One thing to pay attention though it to give it `NET_ADMIN` capability.

You can also have a look at [the search for Kubernetes in issues on this repo](https://github.com/qdm12/gluetun/search?q=kubernetes&type=issues), feel free to ask/comment there if anyone using K8s can share his config. Thanks!
